src/main/java/com/gmail/goosius/siegewar/utils/SiegeWarDistanceUtil.java

```diff
@@ -40,6 +48,46 @@ public static boolean isLocationInActiveSiegeZone(Location location) {
 		return false;
 	}
 
+	public static boolean isTownBlockInActiveSiegeZone(TownBlock townBlock) {
+		World world = Bukkit.getWorld(townBlock.getWorld().getName());
+		int townBlockSize = TownySettings.getTownBlockSize();
+		int x = (townBlock.getX() * townBlockSize) + (townBlockSize /2);
+		int y = 0;
+		int z = (townBlock.getZ() * townBlockSize) + (townBlockSize /2);
+		Location locationOfTownBlock = new Location(world, x, y, z);
+		return isLocationInActiveSiegeZone(locationOfTownBlock);
+	}
+
+	/**
+	 * Get all townblocks inside a siegezone
+	 *
+	 * @param siege the siege
+	 *
+	 * @return all townblocks within the siege zone
+	 */
+	public static Set<TownBlock> getTownBlocksInSiegeZone(Siege siege) {
+		Set<TownBlock> result = new HashSet<>();
+		World siegeBannerWorld = siege.getFlagLocation().getWorld();
+		int siegeBannerLocationX = (int)siege.getFlagLocation().getX();
+		int siegeBannerLocationZ = (int)siege.getFlagLocation().getZ();
+		int townBlockLocationX;
+		int townBlockLocationZ;
+		int townBlockSize = TownySettings.getTownBlockSize();
+		int siegeZoneRadius = SiegeWarSettings.getWarSiegeZoneRadiusBlocks();
+
+		for(Map.Entry<WorldCoord, TownBlock> mapEntry: TownyUniverse.getInstance().getTownBlocks().entrySet()) {
+			if(mapEntry.getKey().getBukkitWorld().equals(siegeBannerWorld)) {
+				townBlockLocationX = (mapEntry.getKey().getX() * townBlockSize) + (townBlockSize /2);
+				townBlockLocationZ = (mapEntry.getKey().getZ() * townBlockSize) + (townBlockSize /2);
+				if(MathUtil.distance(siegeBannerLocationX, townBlockLocationX, siegeBannerLocationZ, townBlockLocationZ) < siegeZoneRadius) {
+					result.add(mapEntry.getValue());
+				}
+			}
+		}
```

Comment:
I believe this method will be unused now?

Reply:
Yes. Will remove.